docs/ppp/pitch-23112022.md

```diff
@@ -0,0 +1,91 @@
+## The project
+
+**10101** (speaks, **Ten** **Ten** **One**). One App, All Things Bitcoin!
+
+10101 is a non-custodial Lightning wallet infused with financial services such as CFD trading, sports betting and savings.
+
+## The team
+
+We are a group of 5 developers who initially built [ItchySats](https://itchysats.network/) together. Now we are bringing non-custodial financial services to Lightning onto your phone.
+
+- bonomat, aka Philipp
+
+  - [https://github.com/bonomat](https://github.com/bonomat)
```

Comment:
I feel this section is quite dominant because all these links are there, maybe we can hide the links and use text?